roc/log_test.go

```diff
@@ -91,6 +93,80 @@ func TestLog_Func(t *testing.T) {
 	}
 }
 
+func TestLog_Message(t *testing.T) {
+	SetLogLevel(LogTrace)
+	defer SetLogLevel(defaultLogLevel)
+
+	tw := makeTestWriter()
+	defer close(tw.ch)
+
+	SetLoggerFunc(func(msg LogMessage) {
+		if msg.Level == LogTrace {
+			msgLevel := strconv.Itoa(int(msg.Level))
+			msgLine := strconv.Itoa(msg.Line)
+			msgPid := strconv.Itoa(int(msg.Pid))
+			msgTid := strconv.Itoa(int(msg.Tid))
+			msgTime := strconv.Itoa(int(msg.Time))
+			byteMsg := msgLevel + ":" + 
```

Comment:
I don't think we need to convert msg to string and then from string to msg in this test. I guess we can just use `ch := make(chan LogMessage, 1)` here